packages/ndla-ui/src/Frontpage/FrontpageMenuPortal.tsx

```diff
@@ -72,10 +71,14 @@ const StyledModalWrapper = styled.div<ModalWrapperProps>`
                 opacity: 0;
                 background: ${colors.brand.light};
                 transform: translate(
-                    calc(-100vw + ${props.elementRect.fromX}px),
-                    calc(-50vh + ${props.elementRect.fromY}px)
+                    calc(
+                      -100vw + ${props.elementRect ? props.elementRect.fromX : ''}px
```

Comment:
Hadde det gått an å bruke ?.
```suggestion
                      -100vw + ${props.elementRect?.fromX || ''}px
```

Reply:
For å bruke optional chaining syntaksen trenger vi typescript 3.7. Jeg bumpa opp typescript fra 3.3.3333 til 3.5.3, men klarte ikke å bumpe den mer uten å ødelegge masse greier i de forskjellie pakkene. Tenker at bump av typescript kan gjøres separat siden det blir en stor jobb i seg selv.

Reply:
Enig.